Documentation/templates/colors/literals-swift4.md

```diff
@@ -29,7 +29,7 @@ Note: if you use `enumName: UIColor` (or `NSColor` on macOS) then the color cons
 **Extract:**
 
 ```swift
-extension ColorName {
+internal extension ColorName {
   /// 0x339666ff (r: 51, g: 150, b: 102, a: 255)
   static let articleBody = #colorLiteral(red: 0.2, green: 0.588235, blue: 0.4, alpha: 1.0)
```

Comment:
1) Why isn't this one `internal` like with other templates?

Reply:
Because I already incorporated the swift4 fixes from the swift5 PR

Reply:
Mmmh ok but then only this one is different from the other, like `colors/swift3` (the non-literal one) doesn't have the same structure and balance of internal present it not (probably because the literals template declare them in an extension instead of directly in the type declaration?)

Then why do we have this inconsistency in swift4 vs literals-swift4 too (extension vs not)? Not very consistent of us 😅

Reply:
Uhm the literals one is so that you can have `UIColor.myRedStuff` if you set the type name to UIColor.

The non-literal version uses struct, with its own properties. It can never be defined as an extension on an existing type such as `UIColor`

Reply:
Ah right. Might be worth make the non-literal one accept to be used as an UIColor extension at one point just got the sake of consistency, but well…

Reply:
Like I said, we can't do that, as it's a struct with a `rgbaValue` and `color` stored properties.

It'd be weird if we somehow retooled the template to allow it, as you'd then use it as:
```swift
UIColor.articleBody.color
```

Reply:
Ah good point!